src/components/about/about.scss

```diff
@@ -0,0 +1,27 @@
+.outer-container {
```

Comment:
When I launch this, I'm seeing several styling issues we need to address and then there's probably some simplification we can do in the code here. Let's schedule a Zoom chat to mob on this together!